test/experimental/test_transforms_with_asset.py

```diff
@@ -169,6 +170,21 @@ def test_builtin_pretrained_sentencepiece_processor(self):
         ref_results = [13, 1465, 12824, 304, 24935, 5771, 3776]
         self.assertEqual(spm_transform(test_sample), ref_results)
 
+    def test_sentencepiece_with_dataloader(self):
+        sp_model_path = download_from_url(PRETRAINED_SP_MODEL['text_bpe_25000'])
+        spm_processor = sentencepiece_processor(sp_model_path)
+        _path = os.path.join(self.project_root, '.data', 'text_bpe_25000.model')
+        os.remove(_path)
+        example_strings = ['the pretrained spm model names'] * 64
+        ref_results = torch.tensor([[13, 1465, 12824, 304, 24935, 5771, 3776]] * 16, dtype=torch.long)
+
+        def batch_func(data):
+            return torch.tensor([spm_processor(text) for text in data], dtype=torch.long)
+
+        dataloader = DataLoader(example_strings, collate_fn=batch_func, batch_size=16)
```

Comment:
[By default DataLoader doesn't do multiprocessing](https://pytorch.org/docs/master/data.html?highlight=dataloader#torch.utils.data.DataLoader). Set the number of workers to a non-zero value (1 is enough to exercise serialization).